### ISSUE TITLE
Trakt.tv in the supported Plex Appstore

### ISSUE THREAD

**Issue by NicoLeOca** [trakt/Plex-Trakt-Scrobbler#454]
Hi @fuzeman 

Is there any procedure to have your plugin available in the official Plex Appstore?
Have you tried to submit it to the Plex team maybe?

It would be interesting to have it on the Nvidia Shield for instance

Thanks for this Plugin and thanks for keeping it up update after update

**Comment by edgard** [comment]
Yeah, it's the only thing actually making me have a separate htpc server instead of just using the Shield. Please the dream come true! :D

**Comment by dgw** [comment]
This can be closed. Plex are dropping the plugin directory entirely: https://www.plex.tv/blog/subtitles-and-sunsets-big-improvements-little-housekeeping/
